### PR TITLE
Remove unneeded torchvision/audio deps

### PR DESCRIPTION
**Pull request by wconstab**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #250

### CODE REVIEW

File: .github/workflows/unit_test_4gpu.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ jobs:
         run: python -m pip install --upgrade pip
       - name: Install dependencies
         run: |
-          pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu121
```
Comment (tianyu-l) [inline .github/workflows/unit_test_4gpu.yaml:36]: I think we need to keep `torch` and just remove `torchvision` and `torchaudio`.
`pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121`

File: .github/workflows/unit_test_cpu.yaml [hunk 2]
```diff
@@ -33,7 +33,6 @@ jobs:
         run: python -m pip install --upgrade pip
       - name: Install dependencies
         run: |
-          pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu121
           python -m pip install -r requirements.txt
```
Comment (tianyu-l) [inline .github/workflows/unit_test_cpu.yaml:37]: ditto
`pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121`

File: .github/workflows/unit_test_4gpu.yaml (hunk 1 above)
Comment (wconstab) [reply .github/workflows/unit_test_4gpu.yaml:36]: haha. yea, my bad

**Comment by awgu** [comment]
Can this PR be landed?

**Comment by wconstab** [comment]
sorry, landing now

**Pull request merged**